oio/cli/admin/cluster.py

```diff
@@ -56,8 +64,109 @@ def take_action(self, parsed_args):
                 addr = srv['addr']
                 up = tags.get('tag.up', 'n/a')
                 score = srv['score']
-                results.append((srv_type, addr, volume, location,
-                                slots, up, score))
+                if not local or addr.find('127.0.0.1') != -1:
```

Comment:
@Crashdown59 : The architecture part aware of local service is the registration point, i.e. the proxy. Whatever, knowing this is not the client's job. Listing the local services has a dedicated route. please consider reading https://github.com/open-io/oio-sds/wiki/OpenIO-Proxyd-API-Reference#list-local-services :)